src/util/svgUtils/types.ts

```diff
@@ -0,0 +1,99 @@
+export type SVGElementAttributeTypes = {
```

Comment:
This list should directly reflect https://developer.mozilla.org/en-US/docs/Web/SVG/Element. If there are things that are no longer on that list we can probably clean the list up a bit

Reply:
Yes i would clean it, i was planning to do an update after i received some feedback!